### DISCUSSION TITLE
Themes by:DJRoby

### DISCUSSION THREAD

**Discussion by DJRoby19** [mathoudebine/turing-smart-screen-python#218] [Themes]
Sharing my first job.

![preview](https://user-images.githubusercontent.com/132849164/236699972-54707d85-2f0b-4123-aa60-1a3361eb6d12.png)

[Ryzen.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11415896/Ryzen.zip)

**Comment by Onslau** [comment]
Great Theme

**Reply by DJRoby19** [reply]
Thank you

**Comment by Onslau** [comment]
![ez panel](https://user-images.githubusercontent.com/132712795/236700489-f902dbbd-ab16-4695-92e2-df9f2ec2beeb.jpg)

**Comment by Onslau** [comment]
![rog 4](https://user-images.githubusercontent.com/132712795/236700504-9a984a5f-d84f-42c3-a427-6864846d2685.jpg)

**Comment by Onslau** [comment]
![Gaming0_generated](https://user-images.githubusercontent.com/132712795/236700519-93461b5a-1200-42af-a27d-ee00c330b82c.jpg)
if you more into this then you can complete my ones

**Comment by DJRoby19** [comment]
Here is another Turing Adapted Theme shared.....
I hope you enjoy....

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/132849164/1a53d77d-8d49-41a2-a494-b8f728f20099)


[Rog.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11436217/Rog.zip)

PS:
Onsau I'm looking at adapting one of your backgrounds...

**Reply by Onslau** [reply]
great just grab one of mine if you have any ideas for overlays share it with me  which backround style etc , if i have time i do make them then !

**Reply by Onslau** [reply]
btw i am using this XD matches perfect to my pc https://youtube.com/shorts/PRcK_B7erC8

**Reply by Onslau** [reply]
if you see it you know what i mean xD

**Comment by Onslau** [comment]
![ironman ui](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/950c3d81-f66a-450b-b7db-14b498e833d1)
have fun

**Reply by Onslau** [reply]
CREATED 4 of them

**Comment by DJRoby19** [comment]
Here is another theme in collaboration with user Onsau who created the backgrond

![screencap](https://github.com/mathoudebine/turing-smart-screen-python/assets/132849164/f1afb2a3-4a3b-43d2-aab2-b70d897e0a76)

[Cyberdemon.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11471974/Cyberdemon.zip)

**Reply by Onslau** [reply]
HOLY ............................ thats nice awesome work!

**Reply by Onslau** [reply]
how do i convert it to a usable theme for the screen you coding it in yaml and the file type is data according to the folders

**Comment by Onslau** [comment]
My next will be some Hacker Backround

**Comment by Onslau** [comment]
![ROBMENTUN](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/505dd8c5-f558-435e-957d-943896a548fe)

Hope you like this one just an idea  Robmentum Hacker overlay

**Reply by Onslau** [reply]
sry iam to fast i need just 20 mins for each xD its just my hobby Pc building and creative things xD ; You can put names away brush overit u know its just for you and me and youre awesome work !

**Reply by Onslau** [reply]
And my Orginal Name is Onslau or Momentum had forgot an L XD

**Reply by DJRoby19** [reply]
> sry iam troppo veloce ho bisogno di soli 20 minuti per ogni xD è solo il mio hobby per costruire PC e cose creative xD ; Puoi mettere via i nomi spazzolandoci sopra, sai che è solo per te e per me e sei un lavoro fantastico!

No instead I would say to leave the names

**Reply by Onslau** [reply]
well okay , with name it looks bit cooler , i look forward next days i try to make more themes need some break for ideas xD Brain got smoking  xD I had to setup server last days mod some things in asetto corsa do some stuff here haha but i love youre work and this community

**Comment by DJRoby19** [comment]
Here's another Nvidia-style theme for enthusiasts... 

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/132849164/557f2d08-815d-42ed-b2f0-b51978033346)

[Nvidia.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11523106/Nvidia.zip)

Enjoy

**Reply by Onslau** [reply]
great job Roby can you do this in red blue  and green , looks so awesome !

**Reply by DJRoby19** [reply]
Thank you dear

**Reply by Onslau** [reply]
can you explain me how to use this theme  on 3.5 screen and where to put it in

**Comment by DJRoby19** [comment]
Using the software
turing-smart-screen-python
you have to insert it inside the res/themes/ folder and insert it inside with all the folder 
Then open the configurator.py and choose the Nvidia theme that's all

**Reply by Onslau** [reply]
ah okay thats the problem your on linux and coding with phyton iam on windows 11 , we need a converter , my format is data and yours is yaml i cant use yours and you cant use my xD

**Reply by DJRoby19** [reply]
Exact,
but you can use turing-smart-screen-python on windows 11

**Reply by Onslau** [reply]
can you explain how to use it i mean a short tutorial  with pictures etc will be nice

**Reply by DJRoby19** [reply]
Meanwhile I'll explain quickly in words, then when I have time I'll take pictures.
First you need to install the python3 libraries,
second unzip the folder
inside the folder open the terminal for example if you want to configure a theme do
python3 configure.py and you are presented with the configuration screen.
to start do python3 main.py
to edit do python3 theme-editor.py and the name of the theme you want to edit.
that's all

**Reply by Onslau** [reply]
thanks alot

**Comment by DJRoby19** [comment]
Onslau I thought the moderator mathoudebine liked my works.
but I haven't seen any comments from him

**Reply by Onslau** [reply]
hmmm he has commentet on my ones idk why not by youre works they are better then mine i do simply overlays and configure them per hand this config takes 2-3 hrs to perfect it like the ironman overlay  i came from goverlay there it was better to config but it was trash after 6 months it simply bricked mad needs to make it more simple for win and phyton users and more stuff like gauges logos etc and and screen to take them in place like goverlay watch this there you see what i mean

**Reply by Onslau** [reply]
https://www.youtube.com/watch?v=ALjFNrM5tbs

**Reply by Onslau** [reply]
![Bild_2023-05-24_015655462](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/2aa6e3db-4dbd-4a0d-aeec-adcea6137d36)
like this

**Reply by Onslau** [reply]
if he get something like this to work it will be better then any stat screen out there

**Comment by Onslau** [comment]
![gtgtg](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/4c47c241-6762-4763-8491-2feb30599261)

**Comment by Onslau** [comment]
![hh](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/911139f2-0a92-4746-8cc1-845ca1c1c846)

**Reply by Onslau** [reply]
![Bild_2023-05-24_034534607](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/f8303799-af17-48bc-9fb2-eb2504cec4e7)
getting this error

**Reply by Onslau** [reply]
error message in english -- 
Start error - the parameter must be positive and less than the altitude. parameter name y

**Reply by Onslau** [reply]
2Hrs of work and perfect it got most out off all bars gauges stats etc and its undefined so everyone can add cpu name and gpu name into gauges or anywhere else hope you can help with this error

**Comment by DJRoby19** [comment]
New version with Radial
Working since Release 3.0.0


![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/132849164/2c4a146b-9c4b-45af-9d1a-5b5d3f802151)


[Ryzen.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12134909/Ryzen.zip)

**Comment by mathoudebine** [comment]
Thanks for your work @DJRoby19 and @Onslau you had a powerful collaboration here! I did not see earlier because i don't receive notifications for discussions

**Reply by mathoudebine** [reply]
Can i add them to the app? You will be credited with your name and a link to your Github profile if it's okay for you!

**Reply by DJRoby19** [reply]
Sure dear, I would be happy to thank you

**Reply by DJRoby19** [reply]
I saw that in the end you didn't include my themes in the project, too bad, I hope you will

**Comment by rahulramirocavalcante-netizen** [comment]
u can make pain them for 3.5?
